ci-operator/step-registry/openshift/microshift/infra/iso-boot/openshift-microshift-infra-iso-boot-commands.sh

```diff
@@ -7,12 +7,6 @@ IP_ADDRESS="$(cat "${SHARED_DIR}"/public_address)"
 HOST_USER="$(cat "${SHARED_DIR}"/ssh_user)"
 INSTANCE_PREFIX="${HOST_USER}@${IP_ADDRESS}"
 
-# May need to update openshift-microshift-infra-aws-ec2-commands.sh to
-# open other ports if changing these settings.
-SSH_EXTERNAL_BASE_PORT=7000
-API_EXTERNAL_BASE_PORT=7100
-LB_EXTERNAL_BASE_PORT=7500
```

Comment:
These variables are still being passed to the command at the end of this file when it runs the remote part of the boot process. We need to keep them until we can update that script to not need them.

Reply:
This one slipped from a different PR, fixed it.